package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "cypress-log-to-output",
-  "version": "1.1.2",
+  "type": "module",
```

Comment:
is this change required for 10.0?

Reply:
I think so. I added it so that I could use ES `import` instead of `require`.

Reply:
@flotwig Wanna maybe deprecate in favor of https://github.com/archfz/cypress-terminal-report ?

Reply:
I also need the v10 support!